metaworld/policies/action.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+class Action:
```

Comment:
This is a namespace, dataclass, or namedtuple depending on your intended usage.

https://docs.python.org/3/library/collections.html#collections.namedtuple
https://docs.python.org/3/library/types.html#types.SimpleNamespace

Note: You an also use dataclasses for this, but they are only available for Python 3.7+, which is a little too cutting-edge for a broadly-used library.